.ci/lib.sh

```diff
@@ -153,6 +153,38 @@ function build() {
 	build_version "${github_project}" "${make_target}" "${version}"
 }
 
+# Build the artifact tarball using kata-deploy build scripts then uncompress
+# it.
+#
+# Parameters:
+#	$1 - artifact name. For example, qemu, qemu-tdx, rootfs-image,
+#	     and so on.
+#
+# Environment variables:
+#       KATA_BUILD_CC - adjust the make targets if building for Confidential
+#			Containers.
+#	DESTDIR - where to uncompress the tarball, by default is '/'.
+#
+function build_static_artifact_and_install() {
+	local artifact="${1:-}"
+	local destdir="${DESTDIR:-/}"
+	local make_target="${artifact}-tarball"
+	local tarball="kata-static-${artifact}.tar.xz"
+
+	# Use different target for Confidential Containers.
+	if [ "${KATA_BUILD_CC:-no}" == "yes" ]; then
+		make_target="cc-${make_target}"
+		tarball="kata-static-cc-${artifact}.tar.xz"
+	fi
+
+	clone_katacontainers_repo
+
+	pushd "$katacontainers_repo_dir" >/dev/null
+	sudo -E PATH=$PATH make "$make_target"
```

Comment:
Just a note for my future reference. I was slightly confused by how this was working as it runs in the top level kata-containers dir and I was expecting us it to need to be in `tools/packaging/kata-deploy/local-build`. Fabiano kindly pointed out that this make file is import in the top-level one: https://github.com/kata-containers/kata-containers/blob/08d230c9402fc0621321c6f40d8a39a8edc93e8a/Makefile#L27